common/src/main/java/gov/cms/ab2d/common/service/AttestationService.java

```diff
@@ -0,0 +1,11 @@
+package gov.cms.ab2d.common.service;
+
+import gov.cms.ab2d.common.model.Attestation;
+import gov.cms.ab2d.common.model.Contract;
+
+public interface AttestationService {
+
+    Attestation saveAttestation(Attestation attestation);
```

Comment:
I think we probably want a single method in this service class that deals with the attestation logic. It would accept a `Sponsor`, `Contract`, and a nullable `Date` as parameters and either update an existing attestation record or create a new one.

Reply:
I think in the case of the attestation report, there should already be a sponsor associated with a contract since we only process contracts currently in the system, and the contract would have an attestation, so it would just need to be updating the attestation date.

Reply:
OK